android/pytorch_android/src/main/cpp/pytorch_jni.cpp

```diff
@@ -582,8 +582,11 @@ class PytorchJni : public facebook::jni::HybridClass<PytorchJni> {
     return makeCxxInstance(modelPath);
   }
 
-  PytorchJni(facebook::jni::alias_ref<jstring> modelPath)
-      : module_(torch::jit::load(std::move(modelPath->toStdString()))) {}
+  PytorchJni(facebook::jni::alias_ref<jstring> modelPath) {
+    at::globalContext().setQEngine(at::QEngine::QNNPACK);
```

Comment:
@supriyar does this call have any overhead (like initializing some quantization engine resource) other than setting a global enum?

Reply:
No overhead. It just sets the qengine flag.